robotbase/src/main/java/com/team2357/frc2022/commands/ClimberSetPivotCommand.java

```diff
@@ -0,0 +1,34 @@
+package com.team2357.frc2022.commands;
+
+import javax.xml.namespace.QName;
+
+import com.team2357.frc2022.subsystems.ClimberSubsystem;
+import com.team2357.lib.commands.CommandLoggerBase;
+
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+
+/**
+ * Command to set the pivot of the pivoting arms
+ * 
+ * @category Climber
+ */
+public class ClimberSetPivotCommand extends CommandLoggerBase {
+    private ClimberSubsystem m_climbSub;
+    private DoubleSolenoid.Value m_value;
+
+    ClimberSetPivotCommand(ClimberSubsystem climbSub, DoubleSolenoid.Value value) {
```

Comment:
Let's not expose DoubleSolenoid.Value as an external interface to the subsystem here. Let's make more meaningful values. It's too hard to remember what's a forward/reverse, and extend/contract value.